service/model_manager/model_list.py

```diff
@@ -1,47 +1,75 @@
+import asyncio
 import os
 import folder_paths
 import server
 from aiohttp import web
 import hashlib
+import shelve
+import threading
+import os
 
-comfy_path = os.path.dirname(folder_paths.__file__)
+# The path to the file where file_hash_dict will be saved
+FILE_HASH_DICT_PATH = os.path.join(os.path.dirname(__file__),"../../hash/file_hash_dict")
```

Comment:
should we use 
`from __init__ import workspace_path`
then path.join(workspace_path, 'hash') in case future dir changes,
its optional but just something to keep cautious about